Mathlib/RingTheory/Radical.lean

```diff
@@ -0,0 +1,93 @@
+/-
+Copyright (c) 2024 Jineon Back and Seewoo Lee. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jineon Baek, Seewoo Lee
+-/
+import Mathlib.RingTheory.UniqueFactorizationDomain
+
+/-!
+# Radical of an element of a unique factorization normalization monoid
+
+This file defines a radical `rad(a)` of an element `a` of a unique factorization normalization
```

Comment:
```suggestion
This file defines a radical of an element `a` of a unique factorization normalization
```
I suggest removing this, because the Lean name is `radical`.